### PR TITLE
[ESS][8.12&8.13] Updating list of available rule actions

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3004

Updated description for the `action_type_id` param on the following pages:

- [Create rule](https://security-docs_bk_4935.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#actions-object-schema)
- [Update rule](https://security-docs_bk_4935.docs-preview.app.elstc.co/guide/en/security/master/rules-api-update.html#actions-object-schema-update)
- [Bulk rule actions](https://security-docs_bk_4935.docs-preview.app.elstc.co/guide/en/security/master/bulk-actions-rules-api.html#actions-object-schema-bulk)

### CODE REVIEW

File: docs/detections/api/rules/rules-api-bulk-actions.asciidoc [hunk 1]
```diff
@@ -376,9 +376,22 @@ These fields are required when calling `PUT` to modify the `actions` object:
 be:
 
 * `.slack`
+* `.slack_api`
 * `.email`
+* .index`
```
Comment (nastasha-solomon) [inline docs/detections/api/rules/rules-api-bulk-actions.asciidoc:381]:
```suggestion
* `.index`
```

File: docs/detections/api/rules/rules-api-update.asciidoc [hunk 2]
```diff
@@ -392,9 +392,22 @@ These fields are required when calling `PUT` to modify the `actions` object:
 be:
 
 * `.slack`
+* `.slack_api`
 * `.email`
+* .index`
```
Comment (nastasha-solomon) [inline docs/detections/api/rules/rules-api-update.asciidoc:397]:
```suggestion
* `.index`
```

**Review by e40pud — APPROVED** [review]
LGTM, thank you!

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**